edx_sysadmin/git_import.py

```diff
@@ -2,31 +2,31 @@
 Provides a function for importing a git repository into the lms
 instance when using a mongo modulestore
 """
-
+# pylint: disable=wrong-import-order
```

Comment:
I saw a lot of similar linting issues in one of our other xBlock (edx-sga) and fixed them in https://github.com/mitodl/edx-sga/pull/326. 

There have been some changes in `edx-lint` the package we use for customized linting along with pylint. We should update our `pylintrc` with `edx-lint` here too but that should be done in a separate PR, I'll try to create a ticket for that and maybe a PR.